src/ts/components/Welcome.tsx

```diff
@@ -0,0 +1,75 @@
+import t from '../services/i18n'
+import * as React from 'react'
+import { Image, Button } from 'semantic-ui-react'
+import styled from 'styled-components'
+import { Link } from 'react-router-dom'
+import { TERM_SERVICE_ROUTE } from '../constants/routes'
+import { IAppState } from '../background/store/all'
+import { connect } from 'react-redux'
+import { saveSettings } from '../background/store/settings'
+import { ThemeTypes } from './styles/themes'
+
+interface IWelcomeProp extends StateProps, DispatchProps {}
+
+interface IHomeState {
+  color: string,
+  theme: ThemeTypes
+}
+
+class Welcome extends React.Component<IWelcomeProp, IHomeState> {
+
+  state: IHomeState = {
+    color: 'red',
+    theme: 'light'
+  }
+
+  handleChangeColor = () => {
+    console.log('Color is changed to %s', this.state.color)
+    this.props.saveSettings({ ...this.props.settings, color: this.state.color })
+  }
+
+  handleChangeTheme = () => {
+    this.props.saveSettings({ ...this.props.settings, theme: this.state.theme })
+  }
+
+  render () {
+    return (
+       <LandingContainer>
+          <Image src='/assets/logo-3-x.svg' size='tiny' />
+          <Image src='/assets/icon-dots.svg' size='small' />
+          <div>
+            {t('welcome')}
+            <div>Current color: {this.props.settings.color}</div>
+            <Button onClick={this.handleChangeColor}>Change color to red</Button>
+
+            <div>Current theme: {this.props.settings.theme}</div>
+            <Button onClick={this.handleChangeTheme}>Change them to light</Button>
+
+          </div>
+          <Link to={TERM_SERVICE_ROUTE}>Term</Link>
+        </LandingContainer>
+    )
+  }
+}
+
+const LandingContainer = styled('div')`
```

Comment:
should we `welcomeContainer `?

Reply:
The benefit is obviously it's more explicit. The downside is we may end up with long names. Do we suffix Container as XXXContainer and Simple Component as YYYComponent? UI libraries seem not doing that. e.g., semantic-ui-react just uses Button, Card, List, Grid, etc. 
I don't mind either way as long as we do that consistently

Reply:
this container is HOC for style container within a component which i think should have the consistent prefix with the component name. It is different with component's `container`.